etc/kayobe/inventory/group_vars/controllers/firewall

```diff
@@ -0,0 +1,375 @@
+    port: 5000
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for keystone_external"
+  - interface: "{{ public_net_interface }}"
+    port: 5601
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for kibana_external"
+  - interface: "{{ public_net_interface }}"
+    port: 6080
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for nova_novncproxy_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8000
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for heat_api_cfn_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8004
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for heat_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8774
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for nova_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8775
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for nova_metadata_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8776
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for cinder_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8780
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for placement_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 8786
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for manila_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 9093
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for prometheus_alertmanager_external"
+  - interface: "{{ public_net_interface }}"
+    port: 9292
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for glance_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 9511
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for magnum_api_external"
+  - interface: "{{ public_net_interface }}"
+    port: 9696
+    proto: tcp
+    destination: "{{ public_net_name | net_vip_address }}"
+    comment: "hint: used by 'haproxy' for neutron_server_external"
+  - interface: "all"
+    port: 111
+    proto: udp
+    comment: "hint: used by 'systemd'"
+  - interface: "all"
+    port: 4789
+    proto: udp
+    comment: "hint: used by 'unknown'"
```

Comment:
is this VXLAN, I think that might be tunnel net only?

Reply:
I've restricted this to the tunnel net only in https://github.com/RSE-Cambridge/cumulus-kayobe-config/pull/21/commits/56b5b0f781a5d7aec59d55cd41129a7ccd7b348a